scripts/install.ps1.tmpl

```diff
@@ -0,0 +1,76 @@
+{{/* $appversion = "v1.0.3" */}}
+
+{{- $urlamd64 := getArchURL . "windows" "amd64" }} 
+{{- $urlarm64 := getArchURL . "windows" "arm64" }} 
+
+$url = "--not-generated--"
+
+$urlamd64 = "{{$urlamd64}}"
+$urlarm64 = "{{$urlarm64}}"
+$destinationPath = "$env:USERPROFILE\Documents\kl"
```

Comment:
just noticed, `Documents\kl` ?

is there a standard place to put EXEs in windows? in linux its one of the bins, most commonly `/usr/local/bin`